pallets/deitos/src/types.rs

```diff
@@ -55,15 +57,22 @@ pub enum PaymentPlanPeriods {
 #[codec(mel_bound(T: pallet::Config))]
 pub struct InfraProviderDetails<T: pallet::Config> {
     // Price per block
-    pub price_per_block: BalanceOf<T>,
+    pub price_storage_per_block: BalanceOf<T>,
+    /// Total IP Storage
+    pub total_storage: Storage,
+    // Available IP Storage
+    pub reserved_storage: Storage,
```

Comment:
is it `Available` or `in use`?

Reply:
it's in use...

Reply:
Ok cool, the comment just seemed confusing to me.

Reply:
yeap, comment was off ... fixed.